go/vt/sqlparser/parser.go

```diff
@@ -229,6 +230,22 @@ func (p *Parser) SplitStatement(blob string) (string, string, error) {
 	return blob, "", nil
 }
 
+// SplitStatements splits a given blob into multiple SQL statements.
+func (p *Parser) SplitStatements(blob string) (statements []Statement, err error) {
+	tokenizer := p.NewStringTokenizer(blob)
+	for {
```

Comment:
can `parser.SplitStatementToPieces` be reused. This method seems to be only used in the test

Reply:
I initially created this function with the intention of using it, until I realized that some tests were using `ExecuteFetchAsDBA` with non-parseable statements, such as `CHANGE MASTER TO ...`. So I ended up not using it. I still think it's nice to keep? If you disagree I'll remove it.

Reply:
I'm happy to use this function in `schemadiff`, which runs the exact same logic.

Reply:
Sounds good to reuse also for schemadiff then.

Reply:
Now using it. It's a bit orthogonal to the topic of this PR.